lib/build.command.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (Aquifer) {
    * Define the 'build' command.
    */
   Aquifer.cli.command('build')
-  .description('Builds a Drupal site in the /builds/work directory.')
+  .description('construct the framework of the site using contents of custom modules, themes, and projects defined in Drush Make.')
```

Comment:
What if we said "Drupal site codebase" instead of "framework of the site"? I think that might be more clear given that the aquifer project codebase could be understood as a framework as well.